requirements/test.txt

```diff
@@ -0,0 +1,13 @@
+# Requirements for test runs
+
+codecov==2.0.15
+diff-cover==1.0.4
+django-nose==1.4.4
+lazy==1.1
+pytest-cov==2.6.0
+tox==3.1.3
+tox-battery==0.5.1
+
```

Comment:
This is missing the `mock` requirement needed to run unit tests on Python 2.7.

Reply:
Nice catch :)